support-payment-api/src/main/scala/backend/StripeBackend.scala

```diff
@@ -299,7 +299,12 @@ class StripeBackend(
     val stripeAcquisition = StripeAcquisition(data, charge, identityId, clientBrowserInfo)
 
     track(
-      acquisition = AcquisitionDataRowBuilder.buildFromStripe(stripeAcquisition, contributionData),
+      acquisition = AcquisitionDataRowBuilder.buildFromStripe(
+        stripeAcquisition,
+        contributionData,
+        similarProductsConsent =
+          false, // TODO: Work out how to get this from the front end taking account of mobile clients
```

Comment:
Same question as above here - do we want to always set this to false going forward? (So no-one is soft opted in until the checkbox is implemented)